open-in-editor

```diff
@@ -102,6 +102,8 @@ class BaseEditor(object):
             sub_cls = VSCode
         elif "vim" in executable_path:
             sub_cls = Vim
+        elif "o" in executable_path:
```

Comment:
Maybe one day we'll have to worry about this being too promiscuous, but I don't think today is that day :)